Cardanian Snowglobes

```diff
@@ -0,0 +1,9 @@
+{
+    "project": "CardanianSnowGlobes",
+    "tags": [
+        "CardanianSnowGlobes"
+    ],
+    "policies": [
+        "2d1e33d10789b2284788e7e67bd3fd2d07194720bc81f475adf45cbe",
```

Comment:
remove comma,
and add proof of project ownership or dm me on discord

Reply:
action from contributor required